tests/common.js

```diff
@@ -46,6 +54,9 @@ function loadQml(src, opts) {
                   ' is known to be failing. Skipping...');
       return;
     }
+    var div = document.createElement('div');
+    this.div = div;
     itOrig.apply(this, arguments);
+    div.remove();
```

Comment:
Style: `this.div.remove()`.
This is more readable, because this way there is no alias for the variable and one doesn't have to keep in mind that `div` and `this.div` are indeed the same var.